src/libPMacc/include/particles/memory/dataTypes/Particle.hpp

```diff
@@ -139,20 +141,21 @@ struct Particle : public InheritLinearly<typename T_FrameType::MethodsList>
 namespace traits
 {
 
-template<typename T_Key,
-typename T_FrameType
->
-struct HasIdentifier<
-PMacc::Particle<T_FrameType>,
-T_Key
->
-{
-private:
-    typedef T_FrameType FrameType;
-public:
-    typedef typename HasIdentifier<FrameType, T_Key>::type type;
```

Comment:
Offline discussion: `HasIdentifier<deselect<Particle,X>,X>` would not compile in old code because 2nd template param was missing. This PR would introduce this bug.
Semantic should be: `HasIdentifier<Particle, X> == true` `<=>` `particle[X_]` is valid